tgui/packages/tgui/interfaces/PlayerPanel/constant.ts

```diff
@@ -1,8 +1,8 @@
 /**
  * @file
- * @copyright 2021
- * @author Mordent (https://github.com/mordent-goonstation)
+ * @copyright 2024
```

Comment:
Copyright year shouldn't change. It's not a "last touched" it's a "copyright since".